templates/main.html

```diff
@@ -47,8 +47,32 @@ <h3>
             {{/leaders}}
           </ul>
         </div>
-      {{/orgs}}
+      {{/withLeader}}
     </div>
+    {{#noLeader.length}}
+      <h2>Organization(s) with no listed leader</h2>
+      <hr>
+      <div class="orgs">
+        {{#noLeader}}
+          <div class="org">
+            <h3>
+              <a href="https://codein.withgoogle.com/organizations/{{slug}}">
+                {{name}}
+              </a>
+              <p>Task Completed: {{completed_task_instance_count}}</p>
```

Comment:
This should be changed to `Tasks Completed` since it's more than one. The same for line 31.